dotnet/src/Connectors/Connectors.OpenAIV2/Settings/OpenAIPromptExecutionSettings.cs

```diff
@@ -11,15 +11,11 @@
 
 namespace Microsoft.SemanticKernel.Connectors.OpenAI;
 
-/* Phase 06 
-- Drop FromExecutionSettingsWithData Azure specific method
-*/
-
 /// <summary>
 /// Execution settings for an OpenAI completion request.
 /// </summary>
 [JsonNumberHandling(JsonNumberHandling.AllowReadingFromString)]
-public sealed class OpenAIPromptExecutionSettings : PromptExecutionSettings
+public class OpenAIPromptExecutionSettings : PromptExecutionSettings
```

Comment:
@stephentoub, just FYI, this class has to be unsealed to allow the `AzureOpenAITextToAudioExecutionSettings` to inherit it.